edenia/antelope-tools
# Fix/rewards distributions 1090

### Adjust rewards distributions in the differents networks

### What does this PR do?

- Resolve #1090
- The calculation of EOS and Proton has not been changed.
- Fix the amount of votes in Telos, Wax and Libre.
- Refactor and avoid unnecessary iterations when rewards are estimated.

### Steps to test

1. Run the project locally with each network.
1. Go to "Block Produces" page.
1. Check the votes and estimated rewards.

### Screenshots

**WAX**

- For Wax, the calculation is the same as for the eos network, except that the percentage of rewards per block produced is 100%.
![imagen](https://user-images.githubusercontent.com/66583677/210090688-88e24eec-1809-4886-acb0-c35968b83718.png)

**Telos**

- For Telos it's approximated with the formula that considers the current price of the token.
![imagen](https://user-images.githubusercontent.com/66583677/210090762-bbf1042a-d468-4c13-b75f-2a8889040c25.png)

**Libre**

- For Libre it's estimated with the expected number of blocks per day.
![imagen](https://user-images.githubusercontent.com/66583677/210090796-938e6431-6acb-4dec-9342-c2183963f599.png)
